platform/mellanox/mlnx-fw-upgrade.j2

```diff
@@ -152,6 +158,11 @@ function GetAsicType() {
     exit "${EXIT_FAILURE}"
 }
 
+function GetMstDevice() {
+    local MST_DEV="$(ls /dev/mst/*_pci_cr0)"
```

Comment:
@liorghub this will produce undesirable error output in case of missing MST device. Suggest to suppress it. Also suggest to move device verification into this function.

Reply:
@nazariig I have changed my code according to your comments.  Now it is aligned with other functions behaviour in this script.